src/general/OptionsMenu.cs

```diff
@@ -577,29 +586,41 @@ private void SwitchMode(OptionsMode mode)
         switch (mode)
         {
             case OptionsMode.MainMenu:
-            {
-                tutorialsEnabled.Hide();
-                optionsMode = OptionsMode.MainMenu;
-                break;
-            }
+                {
+                    tutorialsEnabled.Hide();
+                    optionsMode = OptionsMode.MainMenu;
+                    break;
+                }
 
             case OptionsMode.InGame:
-            {
-                // Current game tutorial option shouldn't be visible in freebuild mode.
-                if (!gameProperties!.FreeBuild)
-                    tutorialsEnabled.Show();
-                else
-                    tutorialsEnabled.Hide();
+                {
+                    // Current game tutorial option shouldn't be visible in freebuild mode.
+                    if (!gameProperties!.FreeBuild)
+                        tutorialsEnabled.Show();
+                    else
+                        tutorialsEnabled.Hide();
 
-                optionsMode = OptionsMode.InGame;
-                break;
-            }
+                    optionsMode = OptionsMode.InGame;
+                    break;
+                }
 
             default:
                 throw new ArgumentException("Options menu SwitchMode called with an invalid mode argument");
         }
     }
 
+    // Displays the current Resolution in options.
+    private void DisplayResolution()
+    {
+        if (resolution != null)
+        {
+            System.Threading.Thread.CurrentThread.CurrentCulture =
+                System.Globalization.CultureInfo.CreateSpecificCulture("en-UK");
+            resolution.Text = "AUTO" + "(" + GetViewportRect().Size.x.ToString()
```

Comment:
This needs to use the translation system. I suggest key name `AUTO_RESOLUTION` with the English content being `Auto ({0}x{1})` and then formatting the width and height to that string.